App/screens/CredentialOffer.tsx

```diff
@@ -68,7 +68,11 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
   const [pendingModalVisible, setPendingModalVisible] = useState(false)
   const [successModalVisible, setSuccessModalVisible] = useState(false)
   const [declinedModalVisible, setDeclinedModalVisible] = useState(false)
-  const dateFormatOptions = { year: 'numeric', month: 'short', day: 'numeric' }
+  const dateFormatOptions: DateTimeFormatOptions = {
```

Comment:
In a later PR/ticket we should collect these into constants (assuming it makes sense).